src/main/scala/com/databricks/spark/csv/CsvRelation.scala

```diff
@@ -17,6 +17,9 @@ package com.databricks.spark.csv
 
 import java.io.IOException
 
+import org.apache.hadoop.io.{Text, LongWritable}
+import org.apache.hadoop.mapred.TextInputFormat
```

Comment:
please order imports according to Spark style guide: https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide#SparkCodeStyleGuide-Imports